packages/auth-construct/src/construct.ts

```diff
@@ -114,11 +116,13 @@ export class AmplifyAuth
   ) {
     super(scope, id);
 
+    this.name = props.name ?? '';
+
     // UserPool
     this.computedUserPoolProps = this.getUserPoolProps(props);
     this.userPool = new cognito.UserPool(
       this,
-      'UserPool',
+      `${this.name ?? ''}UserPool`,
```

Comment:
Can't we just use `this.name` since we are already assigning a default of '' on line 119?

Reply:
good catch, I just updated it